Makefile

```diff
@@ -395,7 +395,7 @@ build-feature-server-java-docker:
 build-feature-server-dev:
 	docker buildx build --build-arg VERSION=dev \
 		-t feastdev/feature-server:dev \
-		-f sdk/python/feast/infra/feature_servers/multicloud/Dockerfile.dev --load .
```

Comment:
was this used at all for deploying any images? Do you want to ask achal or any of the old maintainers?

Reply:
The only difference in the previous Dockerfile vs Dockerfile.dev was the `COPY . .` instruction.

The old Dockerfile installs only the pypi available feast package, whereas the old Dockerfile.dev installs the pypi + copies the current codebase into the image.

With the changes I made to the Dockerfile, it does not rely on pypi for the feast package and we would only use the current codebase to install feast. Essentially the same as what old Dockerfile.dev was doing.